server/src/domain/repositories/asset.repository.ts

```diff
@@ -90,6 +90,27 @@ export type AssetCreate = Pick<
 > &
   Partial<AssetEntity>;
 
+export type AssetWithoutRelations = {
```

Comment:
Why did you do it like this? Why not just use the Omit<AssetEntity, 'field1' | 'field2' > directly?

Reply:
This is more reusable

Reply:
You can have the result of omit as a type. I meant more of that I didn't understand why you need to deconstruct and reconstruct it into an object instead of just referencing export NewType = Omit<...> directly.

Reply:
Oh, that was because I was messing with ways to get this type without specifying all the fields, but gave up on that. I can change it to use the Omit part directly.